### PR TITLE
Update cuvs to match raft's cutlass changes

### PR DESCRIPTION
**Pull request by vyasr**
Due to the tight integration between cuvs and raft, we need to ensure that cuvs is updated for rapidsai/raft#2503 or builds of cuvs that rely on cloning raft will get an incompatible version of cutlass due to raft's update.

### CODE REVIEW

File: cpp/cmake/thirdparty/get_cutlass.cmake [hunk 1]
```diff
@@ -22,7 +24,7 @@ function(find_and_configure_cutlass)
       CACHE BOOL "Enable only the header library"
   )
   set(CUTLASS_NAMESPACE
-      "cuvs_cutlass"
+      "raft_cutlass"
```
Comment (KyleFromNVIDIA) [inline cpp/cmake/thirdparty/get_cutlass.cmake:27]: Was the change from `cuvs` to `raft` intentional?

Comment (KyleFromNVIDIA) [inline cpp/cmake/thirdparty/get_cutlass.cmake:27]: Was the change from `cuvs` to `raft` intentional?

Comment (vyasr) [reply cpp/cmake/thirdparty/get_cutlass.cmake]: Nope, good catch thanks. The export sets were wrong too, fixed those.

**Review by KyleFromNVIDIA — APPROVED** [review]

**Review by benfred — APPROVED** [review]

**Pull request merged**